NextcloudTalk/NCCallController.m

```diff
@@ -455,7 +455,21 @@ - (NCPeerConnection *)getPeerConnectionWrapperForSessionId:(NSString *)sessionId
         peerConnectionWrapper = [[NCPeerConnection alloc] initWithSessionId:sessionId andICEServers:iceServers forAudioOnlyCall:screensharingPeer ? NO : _isAudioOnly];
         peerConnectionWrapper.roomType = roomType;
         peerConnectionWrapper.delegate = self;
-        // TODO: Try to get display name here
+        
+        // Try to get displayName early
+        NSString *displayName = [self getDisplayNameFromSessionId:sessionId];
+        
+        if (displayName) {
+            [peerConnectionWrapper setPeerName:displayName];
+        } else {
+            // Fallback to userId, when displayName can't be resolved
+            NSString *userId = [self getUserIdFromSessionId:sessionId];
+            
+            if (userId) {
+                [peerConnectionWrapper setPeerName:userId];
+            }
```

Comment:
Not sure about this. 
`userId` might be meaningful in some NC instances, but (e.g. when using LDAP or SAML) the `userId` could be a string of random characters or numbers.

Reply:
I see.. so just remove the fallback?

Reply:
I guess...what do you think? :)

Reply:
Yep, totally agree 👍 Haven't thought about LDAP or SAML in this case.